.editorconfig

```diff
@@ -0,0 +1,23 @@
+# EditorConfig is awesome:http://EditorConfig.org
```

Comment:
No reason to not copy it. Or at least change it to match the kestrel one. When I make new repos I usually copy from an arbitrary repository with the assumption that they are all the same. If this one is better, then you can change the razor one, but they shouldn't differ.